internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -38,6 +38,26 @@ import (
 	"github.com/hacbs-contract/ec-cli/internal/evaluator"
 	"github.com/hacbs-contract/ec-cli/internal/policy"
 	"github.com/hacbs-contract/ec-cli/internal/policy/source"
+	e "github.com/hacbs-contract/ec-cli/pkg/error"
+)
+
+var (
+	AP001 = e.NewError("AP001", "", e.ErrorExitStatus) // TODO: add message and set the exit status
```

Comment:
It looks like error codes are not unique across the application. How would we know if `AP001` came from an error in this file or in `internal/applicationsnapshot/input.go`? Well.. I should say: If a user reports seeing `AP001`, how do we map it to the right place in the code?

Reply:
Hard for the linter to know about that, linters work per-package and cross-package with much more effort. Though existing error prefixes would be picked up, so if we take care to have them unique on a per-package basis (something review would need to catch) we should be okay. Also generating documentation from this, listing all error codes should not be that difficult, and that could ease the review...